oms-global/docker/docker-compose.yml

```diff
@@ -40,9 +40,9 @@ services:
     expose:
       - "5432"
     environment:
-      POSTGRES_DB:       api-gw
-      POSTGRES_USER:     kong
-      POSTGRES_PASSWORD: ${PW_POSTGRES}
+      POSTGRES_DB:       api-gw # yamllint disable-line rule:colons
```

Comment:
if you want to ignore that rule, you can disable it globally

Reply:
nah it's that i like how this one is aligned